Source/IntegrationTests/TestFiles/LitTests/LitTest/stdlibs/StandardLibraries_Errors.dfy.expect

```diff
@@ -4,7 +4,9 @@ Dafny program verifier finished with 1 verified, 0 errors
 Dafny program verifier finished with 1 verified, 0 errors
 StandardLibraries_Errors.dfy(20,29): Error: module DafnyStdLibs does not exist (position 0 in path DafnyStdLibs.Wrappers)
 1 resolution/type errors detected in StandardLibraries_Errors.dfy
-*** Error: Cannot load DafnyStandardLibraries-notarget.doo: --unicode-char is set locally to False, but the library was built with True
+CLI: Error: cannot load DafnyStandardLibraries-notarget.doo: --unicode-char is set locally to False, but the library was built with True
```

Comment:
Any way to remove the ":" between "CLI" and "Error" to avoid colon repetition?

Reply:
There's an existing method:

```
public string ErrorToString(ErrorLevel header, IToken tok, string msg) {
    return $"{tok.TokenToString(Options)}: {header.ToString()}: {msg}";
  }
```

that creates the double colon. We could remove the first one.